src/roswire/ros2/node_manager.py

```diff
@@ -1,30 +1,42 @@
 # -*- coding: utf-8 -*-
 __all__ = ('ROS2NodeManager',)
 
-from typing import Iterator, Mapping
+from typing import Iterator
 import typing
+import attr
```

Comment:
`attr` is a third-party module, so it should be grouped with `loguru`. (See previous version.)